units/src/amount/signed.rs

```diff
@@ -58,9 +58,9 @@ impl SignedAmount {
     /// The maximum value allowed as an amount. Useful for sanity checking.
     pub const MAX_MONEY: SignedAmount = SignedAmount(21_000_000 * 100_000_000);
     /// The minimum value of an amount.
-    pub const MIN: SignedAmount = SignedAmount(i64::MIN);
+    pub const MIN: SignedAmount = SignedAmount(-21_000_000 * 100_000_000);
```

Comment:
If we are going to enforce a lower bound that's not i64::MIN , should that be MIN_MONEY?  Not sure since core doesn't.

Reply:
I have no strong opinion either way on adding a `MIN_MONEY` or leaving as it is.  But lean towards not adding one since there was already discussion of removing `MAX_MONEY`.

Reply:
-1 to adding MIN_MONEY